plugins/woocommerce-admin/client/products/fields/attribute-field/add-attribute-modal.tsx

```diff
@@ -219,31 +246,63 @@ export const AddAttributeModal: React.FC< AddAttributeModalProps > = ( {
 														/>
 													</td>
 													<td className="woocommerce-add-attribute-modal__table-attribute-value-column">
-														<AttributeTermInputField
-															placeholder={ __(
-																'Search or create value',
-																'woocommerce'
-															) }
-															disabled={
-																! formAttr.id
-															}
-															attributeId={
-																formAttr.id
-															}
-															value={
-																formAttr.terms
-															}
-															onChange={ (
-																val
-															) =>
-																setValue(
-																	'attributes[' +
-																		index +
-																		'].terms',
+														{ isEmptyItem(
+															formAttr
```

Comment:
I know that `formAttr` was not added in this PR, but this confused me for a minute as I thought it was a "form attribute."   Would `attribute` be a more accurate name?  Or is there something special about these that is related to the form?